features/web-audio.yml

```
@@ -0,0 +1,190 @@
+name: Web Audio
+description: The Web Audio API creates graphs of audio nodes used to handle sound inputs, synthesize sounds, apply effects, create visualizations, and output to audio devices.
```

Should we name `AudioContext` in the description? Usually we do that early in the description, but saying "Also known as Web Audio API" at the end in this case would be embarrassing, since that's how the feature is usually referred to.

---

OK, let's leave it out for now.

---

Sorry, I'm late to this with a style nits (avoiding "used to").

```suggestion
description: The Web Audio API creates graphs of audio nodes that handle sound inputs, synthesize sounds, apply effects, create visualizations, and output to audio devices.
```